var/spack/repos/builtin/packages/bwa/package.py

```diff
@@ -19,10 +20,20 @@ class Bwa(Package):
             url='https://github.com/lh3/bwa/archive/0.7.12.tar.gz')
 
     depends_on('zlib')
+    depends_on('sse2neon', when='target=aarch64:')
+
+    patch('bwa_for_aarch64.patch', sha256='b77213b16cf8760f01e32f9a0b2cd8988cf7bac48a11267100f703cbd55c4bfd', when='target=aarch64:')
 
     def install(self, spec, prefix):
-        filter_file(r'^INCLUDES=',
-                    "INCLUDES=-I%s" % spec['zlib'].prefix.include, 'Makefile')
+        zlib_inc_path = spec['zlib'].prefix.include
+        sse2neon_inc_path = spec['sse2neon'].prefix.include
```

Comment:
This line will cause installation to crash on all non-aarch64 machines. Put it inside of the if-statement.

Reply:
Got it, fix in https://github.com/spack/spack/pull/17473/commits/3371f66d1fd6bfb2f5fdb9296d907a4dc34fac24